props/saber_fett263_buttons.h

```diff
@@ -110,10 +110,16 @@
 // FETT263_THRUST_ON_NO_BM
 // To enable Thrust On Ignition control but not activate Battle Mode (works with THRUST_ON_PREON only)
 //
+// FETT263_THRUST_LENGTH 50
```

Comment:
I don't really think you need defines for everything.
You don't really need to change it, but too many defines are just going to confuse users and make it harder to keep things backwards compatible. People who really want to tweak things can just fiddle with the code directly.

Reply:
OK, wasn't sure if it made it easier or harder but I've removed.